include/ginkgo/core/matrix/batch_dense.hpp

```diff
@@ -337,6 +337,20 @@ class BatchDense : public EnableBatchLinOp<BatchDense<ValueType>>,
                               make_temporary_clone(exec, b).get());
     }
 
+    /**
+     * Adds `a` scaled by `alpha` to the matrix scaled by `beta`.
+     *
```

Comment:
Maybe add the expression,
```suggestion
     *
     * this = alpha * a + beta * this
     *
```